lib/http_event_store/helpers/parse_entries.rb

```diff
@@ -13,18 +13,19 @@ def call(entries)
       private
 
       def create_event(entry)
-        return nil unless entry['data']
```

Comment:
That's part of https://github.com/arkency/http_event_store/pull/21 - maybe let's move this out to #21